pyzscaler/zia/__init__.py

```diff
@@ -210,3 +211,19 @@ def vips(self):
 
         """
         return DataCenterVIPSAPI(self)
+
+    @property
```

Comment:
I think you may have accidentally duplicated this class attribute. We have the admin_and_role_management attribute defined on line 95. 

Can you just remove lines 216->221 when you resubmit?